### PR TITLE
Addex

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: addex.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                break;
+            default:
+                alert("Неверное значение");
+        }
+    </script>
+
+    <script>
+        let str = "abcde";
+        console.log(typeof str);
+        if (str[0] == "a") {
```
Comment (kathrynowy) [inline addex.html:56]: strict comparison

File: addex.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            alert("Yes")
+        } else {
+            alert("No")
+        }
+    </script>
+
+    <script>
+        let num = "12345";
+        if (num[0] == "1" || num[0] == "2" || num[0] == "1") {
```
Comment (kathrynowy) [inline addex.html:65]: two same conditions

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: addex.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        if (str[0] === "a") {
+            alert("Yes");
+        } else {
+            alert("No");
+        }
+    </script>
+
+    <script>
+        let num = "12345";
+        if (num[0] == "1" || num[0] == "2") {
```
Comment (kathrynowy) [inline addex.html:65]: strict comparison!

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: addex.html (hunk 1 above)
Comment (Kartynnikk) [reply addex.html]: fixed

File: addex.html (hunk 2 above)
Comment (Kartynnikk) [reply addex.html]: fixed

File: addex.html (hunk 3 above)
Comment (Kartynnikk) [reply addex.html:65]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**